P5/Source/Guidelines/en/BIB-Bibliography.xml

```diff
@@ -1203,8 +1206,12 @@ $Id$
           Boucle</title>, <date>1993</date>.</bibl>
       <bibl xml:id="COHQQ-eg-35"><author>Rowling, J. K.</author>
         <title level="a">The Sorting Hat</title>. In <title level="m">Harry Potter and the
-          Sorcerer's Stone</title>, (<date>1999</date>), <biblScope unit="chap">chapter
-          7</biblScope>, <biblScope unit="pp">p.121</biblScope>.</bibl>
+          Sorcerer's Stone</title>, <pubPlace>New York</pubPlace>: <publisher>Scholastic, Inc.</publisher> (<date>1999</date>), <biblScope unit="chap">chapter
+          7</biblScope>, <biblScope unit="pp">p. 121</biblScope>.</bibl>
+      <bibl xml:id="rubin-NDPERSEpc"><author>Rubin, Gayle</author>, <title level="a">The Traffic in Women: 
```

Comment:
Should this work be cited in the bibliography related to the examples, or isn‘t it better within the list of “Works Cited Elsewhere in the Text of these Guidelines”?

Reply:
@HelenaSabel Oh, thank you! I'll move it to the appropriate list.